examples/snap/snap.jl

```diff
@@ -40,16 +40,12 @@ A = LMP(["-screen","none"]) do lmp
         command(lmp, "dump 2 all custom 100 dump.forces fx fy fz")
         command(lmp, "run 0")
 
-        # LAMMPS returns this as double...
-        natoms = Int(get_natoms(lmp))
+        nlocal = extract_global(lmp, "nlocal")
 
         types = extract_atom(lmp, "type", LAMMPS.API.LAMMPS_INT)
```

Comment:
@rohskopf one thing I noticed is that the types are not ordered -- not all type 1 atoms are listed as the first 96 atoms.